docs/kh2/file/type/00battle.md

```diff
@@ -650,17 +732,19 @@ Contains the level-up table for summons and drive forms.
 
 | Offset | Variable Type | Description |
 |--------|---------------|-------------|
-| 0 	 | uint | File type (2)
-| 4 	 | uint | Entry Count
+| 0 	 | uint32 | File type (2)
+| 4 	 | uint32 | Entry Count
 
 ### Fmlv Entry
 
 | Offset | Type  | Description
 |--------|-------|-------------
-| 00     | byte  | First digit is the Form id, second digit is the Form level (e.g. 0x13 is Valor Form Level 3)
-| 01     | byte  | Level of the movement ability in the form (High Jump, Quick Run etc.)
-| 02     | short | Ability obtained through level up - [ITEM/ABILITY LIST](../../dictionary/inventory.md)
-| 04     | int   | EXP needed for level up
+| 00     | uint8 : 0-3 | [Id](#forms)
+| 00     | uint8 : 4-7 | Level
+| 01     | uint8 : 0-3 | Ability Level
+| 01     | uint8 : 4-7 | Anti Rate
```

Comment:
Anti rate and Ability Level are inversed. (Anti = 0-3; Abi = 4-7)

Reply:
What are Ability Levels?

Reply:
Growth Ability levels for each form.
Eg: Valor form's level 1 and 2 have a Lvl 1 High Jump.

Reply:
![grafik](https://user-images.githubusercontent.com/29046361/112736809-bfee7500-8f55-11eb-859f-fb63d74245a1.png)
Am i confusing something?

Reply:
I'll show you Valor data so you can see:
![image](https://user-images.githubusercontent.com/20492864/112747683-3d969d00-8fb7-11eb-8fac-99638625d4cb.png)

And you wrote it right for the tool (Note that the function writes from right to left, so "4,4" are the first 4 bits):
![image](https://user-images.githubusercontent.com/20492864/112747690-47b89b80-8fb7-11eb-86c9-ab6d0701b0d7.png)
![image](https://user-images.githubusercontent.com/20492864/112747697-5010d680-8fb7-11eb-80f5-a43e126d1845.png)
![image](https://user-images.githubusercontent.com/20492864/112747705-5737e480-8fb7-11eb-8e36-b88856bcefcc.png)
![image](https://user-images.githubusercontent.com/20492864/112747710-5c952f00-8fb7-11eb-97c5-7ea088a2a184.png)